ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -50,8 +50,8 @@
 
 
 class UnityEnvironment(BaseEnv):
-    SCALAR_ACTION_TYPES = (int, np.int32, np.int64, float, np.float32, np.float64)
-    SINGLE_BRAIN_ACTION_TYPES = SCALAR_ACTION_TYPES + (list, np.ndarray)
+    _SCALAR_ACTION_TYPES = (int, np.int32, np.int64, float, np.float32, np.float64)
+    _SINGLE_BRAIN_ACTION_TYPES = _SCALAR_ACTION_TYPES + (list, np.ndarray)
```

Comment:
I think these can be removed - SINGLE_BRAIN_ACTION_TYPES is unused, and SCALAR_ACTION_TYPES is only used in _flatten, which is unused.